libs/components/toast/testing/src/toast-fixture.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { Component } from '@angular/core';
 import { TestBed } from '@angular/core/testing';
 import { NoopAnimationsModule } from '@angular/platform-browser/animations';
-import { SkyToastModule, SkyToastType } from '@skyux/toast';
+import { λ1 as SkyToastComponent, SkyToastType } from '@skyux/toast';
```

Comment:
Needed to import the component since it's standalone. Consumers will not have to do this.